File: in_Java/0151 Reverse Words in a String.java
class Solution {
    public String reverseWords(String s) {
        String[] st = s.split(" ");
        String res = "";
        for(int i = st.length-1; i >= 0; i--){
            if(st[i].length() == 0) continue;
            res += st[i] + " ";
        }
        return res.strip();
    }
}

File: in_Python/0151 Reverse Words in a String.py
class Solution(object):        
    def reverseWords(self, s):
        """
        :type s: str
        :rtype: str
        """
        s = s.strip()
        word = s.split(' ')
        words = []
        for w in word:
            if w == '':
                continue
            words.insert(0,w)
        #print(words)
        newwords = ''
        for w in words:
            #ne = self.rev(w)
            newwords += w
            newwords += ' '
        
        return newwords.strip()